docs/application-setup.md

```diff
@@ -9,7 +9,7 @@ Choose the framework to build your application:
 [SAPUI5/OpenUI5](#sapui5)<br>
 [VUE.JS](#vuejs)
 
-## Intitial steps
+## Initial steps
```

Comment:
I was wondering if we maybe can have something better than `initial steps` Maybe `Basic application setup` 

what do you think?

Reply:
You are right. Done.